google-cloud-datastore/src/main/java/com/google/cloud/datastore/AggregationResult.java

```diff
@@ -24,21 +27,52 @@
 /** Represents a result of an {@link AggregationQuery} query submission. */
 public class AggregationResult {
 
-  private final Map<String, LongValue> properties;
+  private final Map<String, Value<?>> properties;
 
-  public AggregationResult(Map<String, LongValue> properties) {
+  public AggregationResult(Map<String, Value<?>> properties) {
     this.properties = properties;
   }
 
   /**
-   * Returns a result value for the given alias.
+   * Returns a result value for the given alias. {@link #getLong(String)} is preferred over this
+   * method, Use {@link #getLong(String)} wherever possible.
    *
    * @param alias A custom alias provided in the query or an autogenerated alias in the form of
    *     'property_\d'
    * @return An aggregation result value for the given alias.
    */
   public Long get(String alias) {
-    return properties.get(alias).get();
+    return getLong(alias);
+  }
+
+  /**
+   * Returns a result value for the given alias.
+   *
+   * @param alias A custom alias provided in the query or an autogenerated alias in the form of
+   *     'property_\d'
+   * @return An aggregation result value for the given alias.
+   */
+  public Long getLong(String alias) {
+    Value<?> value = properties.get(alias);
+    if (value.getType() == DOUBLE) {
+      return ((Double) value.get()).longValue();
+    }
+    return (Long) value.get();
```

Comment:
This may not be a safe cast if more types are supported in the future.
Maybe check for `== LONG` here.

Reply:
addressed as part of [b60995b](https://github.com/googleapis/java-datastore/pull/1067/commits/b60995b0be6813d5f302006b02ae74729d0aa33f).